test/pageobjects/Camera.js

```diff
@@ -40,6 +41,13 @@ class Camera extends BasePage {
     return cameraClasses.includes('onfido-sdk-ui-Overlay-isWithoutHole')
   }
 
+  async verifyOnFidoFooterIsVisible() {
```

Comment:
Onfido doesn't have a capital F 😅 
```suggestion
  async verifyOnfidoFooterIsVisible() {
```